mlflow/cli.py

```diff
@@ -154,7 +154,7 @@ def ui(file_store, host, port, gunicorn_opts):
     """
     # TODO: We eventually want to disable the write path in this version of the server.
     try:
-        _run_server(file_store, file_store, host, port, 1, None, gunicorn_opts)
+        _run_server(file_store, None, host, port, 1, None, gunicorn_opts)
```

Comment:
In `FileStore` if `artifact_root_uri` is `None`, it uses root directory

```
class FileStore(AbstractStore):
    ....

    def __init__(self, root_directory=None, artifact_root_uri=None):
        """
        Create a new FileStore with the given root directory and a given default artifact root URI.
        """
        super(FileStore, self).__init__()
        self.root_directory = root_directory or _default_root_dir()
        self.artifact_root_uri = artifact_root_uri or self.root_directory
        ...
```

Reply:
Can we add a click alias, and update the documentation to indicate that the available schemas? We can also go ahead and update all public-facing docs to use this new parameter.

Reply:
Same with `server`. There, though, we should be clearer about about what the default is ("in the file store" does not make sense). I think it might be reasonable to make this parameter required when using the sqlalchemy store, at least for the `server` CLI. For the `ui` one, we could reuse `mlruns`, which is the existing default for the file store.